src/main/java/com/mcmiddleearth/entities/entities/composite/animation/BakedAnimationTree.java

```diff
@@ -104,15 +135,22 @@ public BakedAnimation getAnimation(String[] path) {
 
     public BakedAnimation getAnimation(BakedAnimationEntity entity) {
         String[] path = new String[] {
-                entity.getMovementType().name().toLowerCase(),
-                entity.getMovementSpeedAnimation().name().toLowerCase(),
-                ActionType.IDLE.name().toLowerCase()
-                //entity.getActionType().name().toLowerCase()
+            entity.getMovementType().name().toLowerCase(),
+            entity.getMovementSpeedAnimation().name().toLowerCase(),
+            getActionType(entity).name().toLowerCase(Locale.ROOT)
```

Comment:
I think this was done on purpose and will break animation lookups if changed?

Reply:
port/magma-1.16 has the same getActionType() method, that's the one that should be used, the feature/rework_animation_frames branch also has one but without the goals implemented in magma